### PR TITLE
Update DynamoDB local download link

### PR DESCRIPTION
**Pull request by peterfotinis**
Current link is returning HTTP status 404 not found. New link copied from https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html

### CODE REVIEW

**Comment by Si1kIfY** [comment]
Wow great minds, have literally just done this locally too and was going to submit a PR

Opened #7 for it

Might just run a forked version until fixed

File: dynamodb/installer.js [hunk 1]
```diff
@@ -11,39 +11,33 @@ var tar = require('tar'),
 var download = function (downloadUrl, installPath, callback) {
     console.log("Started downloading Dynamodb-local. Process may take few minutes.");
     http.get(downloadUrl, function (response) {
```
Comment (Si1kIfY) [inline dynamodb/installer.js]: Worth changing this to `https`?

**Comment by peterfotinis** [comment]
I held off using https to maximise compatibility (eg if someone was using a build server depending on this package which only allowed port 80 or some super weird corporate firewall). I know it's probably unlikely, just did not want to patch it and break things. Perhaps a version bump and change to https would be a better path, as a separate pull request?

**Comment by rehrumesh** [comment]
@AshanFernando @mjzone Please review.
@peterfotinis Thanks for the work... 👍

**Comment by Si1kIfY** [comment]
@peterfotinis could do a https check and run http if it fails?

you're right, should be part of a seperate PR

why is travis-ci failing?

**Pull request merged**